src/wilson/dom.cljs

```diff
@@ -13,12 +13,11 @@
              [:th (capitalize k)]))]
     (into [:tbody]
           (for [row rows]
-            (into [:tr
-                   {:class (row->cls row)}]
+            (into [:tr (or (row->attrs row) {})]
                   (for [k keys]
                     [:td (get row k)]))))])
   ([keys rows]
-   (table keys rows {:row->cls :wilson/row-class})))
+   (table keys rows {:row->attrs {}})))
```

Comment:
`{}` was my solution to every conditional statement returning `nil` - I don't want to return anything if there's no `row->attrs` present, and that's what I was having trouble with